packages/core/useStepper/index.ts

```diff
@@ -0,0 +1,83 @@
+import { computed, ref } from 'vue-demi'
+
+export function useStepper<T extends string>(steps: readonly T[], initial?: T) {
+  const index = ref(steps.indexOf(initial ?? steps[0]))
+  const current = computed(() => steps[index.value])
+  const isFirst = computed(() => index.value === 0)
+  const isLast = computed(() => index.value === steps.length - 1)
+
+  function backTo(step: T) {
+    if (currentStepIsAfter(step))
+      goTo(step)
+  }
+
+  function goTo(step: T) {
+    index.value = steps.indexOf(step)
+  }
+
+  function next() {
+    if (isLast.value)
+      return
+
+    index.value++
+  }
+
+  function previous() {
+    if (isFirst.value)
+      return
+
+    index.value--
+  }
```

Comment:
I somehow prefer these to be computed.

Reply:
Oh, I misunderstand them. Probably having both action and ref, like `next()` and `nextStep.value`, or `goNext()`?

Reply:
Good idea!

Reply:
Done! Though I'm not sure when I would use them, it's nice to have them.